packages/amplify_datastore/example/lib/main.dart

```diff
@@ -115,12 +115,14 @@ class _MyAppState extends State<MyApp> {
       postWithCreatedDate += encoder.convert(element.toJson()) + '\n';
     });
 
+    /*
     (await Amplify.DataStore.query(Post.classType,
-            where: QueryField(fieldName: "post.id")
+            where: QueryField(fieldName: "id")
```

Comment:
This should be changed to Post.ID.ne... as we discussed.

Reply:
ah yes.  changed the code